internal/pkg/manifest/lb_fargate_manifest_test.go

```diff
@@ -58,7 +58,13 @@ count: 1
 #  test:
 #    count: 2               # Number of tasks to run for the "test" environment.
 `
-	m := NewLoadBalancedFargateManifest("frontend", "frontend/Dockerfile")
+	m := NewLoadBalancedFargateManifest(&LBFargateManifestProps{
+		AppManifestProps: &AppManifestProps{
+			AppName: "frontend",
```

Comment:
awesome, I love dropping the prefix "/". I ran into this yesterday, where I couldn't figure out why I kept getting a 503 until David pointed out that i forgot the "/" at the beginning 🤦‍♂️

Reply:
💁‍♂️